### ISSUE TITLE
Nextcloud-Whitelist: false-positive when opening mp3 files in public shared folders

### ISSUE THREAD

**Issue by AndiMb** [crowdsecurity/hub#1131]
**Describe the bug**
The nextcloud whitelist ([parsers/s02-enrich/crowdsecurity/nextcloud-whitelist.yaml](https://github.com/crowdsecurity/hub/blob/master/parsers/s02-enrich/crowdsecurity/nextcloud-whitelist.yaml)) is missing a case when e.g. playing mp3 files in a public shared folder.

**To Reproduce**
Opening a shared folder with mp3 files an play the mp3 files in Nextcloud (v29.0.8) will eventually trigger http-probing.

**part of the line from apache2 log**
`GET /apps/files_sharing/publicpreview/qdrXrZMw2mH2Gzk?fileId=518964&file=%2FILENAME.mp3&c=959e9a59ea8537cad1fc86fc3f18f914&x=500&y=500 HTTP/2.0`

**possible solution to add in nextcloud-whitelist.yaml**
`- evt.Meta.http_status == '404' && evt.Meta.http_verb == 'GET' && evt.Meta.http_path contains '/apps/files_sharing/publicpreview' && evt.Parsed.http_args contains 'fileId=' # 404 error thrown when preview is missing for files in public preview`

**Comment by LaurenceJJones** [comment]
Good point @ThomasCr I didnt see op had different X and Y, I made it from our enterprise support side.

**Comment by LaurenceJJones** [comment]
Merged fix as purposed by @ThomasCr thank you for pointing it out on this issue, I would never of seen it.
